src/renderer/components/editors-toolbar-button.tsx

```diff
@@ -17,7 +17,7 @@ interface ToolbarButtonProps {
 
 abstract class ToolbarButton extends React.PureComponent<ToolbarButtonProps> {
   public static contextType = MosaicWindowContext;
-  public context: MosaicWindowContext;
+  public declare context: MosaicWindowContext;
```

Comment:
Note, these `declare` changes were necessary and follow [the pattern in `@types/react`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/d3a514567da8d4d146dab60eadda33bcd1e1b29c/types/react/v16/index.d.ts#L464-L480):

```ts
/**
 * If using the new style context, re-declare this in your class to be the
 * `React.ContextType` of your `static contextType`.
 * Should be used with type annotation or static contextType.
 *
 * ```ts
 * static contextType = MyContext
 * // For TS pre-3.7:
 * context!: React.ContextType<typeof MyContext>
 * // For TS 3.7 and above:
 * declare context: React.ContextType<typeof MyContext>
 * ```
 *
 * @see https://react.dev/reference/react/Component#context
 */
// TODO (TypeScript 3.0): unknown
context: any;
```